scripts/ovadownload.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "==> Searching for OVA download URL..."
+ova_download_url=$(grep -w ovaDownloadUrl vcd/vcd_test_config.json | tr -d '",' | awk '{print $2}')
+ova_file_name=$(grep -w ovaTestFileName vcd/vcd_test_config.json | tr -d '",' | awk '{print $2}')
```

Comment:
1.) Are you sure you did not forget to update vcd_test_config.json? I can't see "ovaDownloadUrl" and "ovaTestFileName" there. How this should work?
2.) Do we need "ova_file_name"? Can't we get the filename from the URL?
3.) Rename `ovadownload.sh` -> `download-ova.sh` to be more consistent with things like `install-plugin.sh`

Reply:
See my comment above. If we implement the OVA retrieval in the Go `TestMain`, we just use the fields in the configuration file, with more control on what we are doing.

Reply:
Thanks for responses and config sample update.

> Well I used same naming convention which is are '''scripts''' folder. Are we changing all them?

Someone added "install-plugin.sh" and it looks just so much nicer with the dash :)

Reply:
In current repo version there isn't such file :) 
> Someone added "install-plugin.sh" and it looks just so much nicer with the dash :)

Reply:
No script - no issue here anymore :)